.github/workflows/build.yml

```diff
@@ -652,12 +652,18 @@ jobs:
       - name: Verify tarball content
         run: |
           yarn test test/pack.test.ts
+      - name: Release (dry run)
+        if: ${{ github.ref != 'refs/heads/trunk' }}
+        run: |
+          yarn nx release --dry-run
```

Comment:
The dry run needs a GitHub auth token to read GitHub releases if you make those, I believe.

Reply:
I think this should cover it:
![image](https://github.com/user-attachments/assets/ff2c0bd0-2e77-4b7e-9be6-8d489bb7ba19)